utility/gmusic_playlists_to_plex.py

```diff
@@ -87,6 +87,17 @@ def compare(ggmusic, pmusic):
     elif title == pmusic.title:
         return [pmusic]
 
+songs = mc.get_all_songs()
+def get_songs_info(id_song):
```

Comment:
can you change the variable names to make sense of what is being used.
`id_song` > `songId`
` for e in songs:` > ` for song in songs:`